test/e2e/specs/all-features-confirmation/record/presentation.spec.js

```diff
@@ -50,7 +50,9 @@ var testParams = {
         { title: "Cover Image", value: "3,005", type: "int2", presentation: { type: "url", template: "{{{chaise_url}}}/record/#{{catalog_id}}/product-record:file/id=3005"} },
         { title: "Thumbnail", value: null, type: "int4"},
         { title: "Operational Since", value: "2008-12-09 00:00:00", type: "timestamptz" },
-        { title: "Is Luxurious", value: "true", type: "boolean" }
+        { title: "Is Luxurious", value: "true", type: "boolean" },
+        { title: "json_col", value:'<pre>'+JSON.stringify(null,undefined,2)+'</pre>'},
+        { title: "jsonb_col", value:'<pre>'+JSON.stringify(null,undefined,2)+'</pre>'}
```

Comment:
The expectation here is `expect(el.getAttribute(‘innerHTML’)).toBe(column.value);`. so the entire innerHTM is beign picked with the whitespaces that is beign introduced when the JSON value is stringified with a preetiness factor of 2, using JSON.stringify(value,undefined,2);